velox/tool/trace/OperatorReplayerBase.cpp

```diff
@@ -68,6 +68,13 @@ core::PlanNodePtr OperatorReplayerBase::createPlan() const {
   const auto* replayNode = core::PlanNode::findFirstNode(
       planFragment_.get(),
       [this](const core::PlanNode* node) { return node->id() == nodeId_; });
+  // Leaf node.
```

Comment:
Shall we directly check if it is table scan plan node? Only source node of the non-leaf stage might have different processing? Thanks!

Reply:
>only source node of the non-leaf stage might have different processing? 

Yes

>Shall we directly check if it is table scan plan node? 

Got it, will fix it.